lib/ObjWriter/jitDebugInfo.h

```diff
@@ -0,0 +1,46 @@
+#ifndef JIT_DEBUG_INFO_H
+#define JIT_DEBUG_INFO_H
+
+typedef unsigned int DWORD;
+#define _TARGET_AMD64_ 1
+
+#include "cordebuginfo.h"
+#include "cvconst.h"
+#include "llvm/DebugInfo/CodeView/SymbolRecord.h"
+
+// DIA headers must come after windows headers.
+
+struct DebugLocInfo {
+  int NativeOffset;
+  int FileId;
+  int LineNumber;
+  int ColNumber;
+};
+
+struct DebugVarInfo {
+  int varNumber;
```

Comment:
I don't see the need of 'varNumber' other than asserting it which is doable locally.